validate/validate.h

```diff
@@ -9,6 +9,20 @@ namespace pgv {
 namespace protobuf = google::protobuf;
 namespace protobuf_wkt = google::protobuf;
 
+class UnimplementedException {
```

Comment:
Should this be a std::runtime_error subclass? That way you can provide a `what()` implementation instead of `String`